layouts/shortcodes/keps-data.html

```diff
@@ -0,0 +1,76 @@
+{{/*
+  Datatables is used in the page to enrich the list of KEPs.
+  The stylesheets and scripts required are in `partials/hooks/head-end.html`
+  and `partials/hooks/body-end.html` respectively.
+
+  Note: In order to configure Datatable parameters for changing any behavior, edit
+  the `DataTable` initialization in `partials/hooks/body-end.html`
+
+*/}}
+
+<div>
+  <table id="keps-table" class="table table-hover mt-2 col-md-12 table-sm">
```

Comment:
nit: If someone were to use that shortcode twice in one page, the output would have a duplicated `id`. I think this is low risk low impact.